lab-yana/app/entry.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+require('./scss/reset.scss');
+require('./scss/main.scss');
+
+const angular = require('angular');
+const cowsay = require('cowsay-browser');
+const mooApp = angular.module('mooApp', []);
+
+mooApp.controller('MooController', ['$log', '$scope', MooController]);
+
+function MooController($log, $scope) {
+  $log.debug('MooController');
+
+  let mooCtrl = $scope.mooCtrl = {};
+  mooCtrl.title = 'Cows are the best!';
+
+  mooCtrl.speak = function(input) {
+    $log.debug('mooCtrl.speak');
+    return cowsay.say( { text: input || 'MOOOOOOOO!!!!'} );
+  };
```

Comment:
One of the requirements of the assignment is for you to use the functionalities we created in class, or for you to make up a few of your own. Please add two or more methods to show you have a basic understanding of Angular. 👍